contrib/database/sql/option.go

```diff
@@ -4,30 +4,37 @@ import (
 	"math"
 )
 
-type registerConfig struct {
+type config struct {
 	serviceName   string
 	analyticsRate float64
+	dsn           string
 }
 
-// RegisterOption represents an option that can be passed to Register.
-type RegisterOption func(*registerConfig)
+// Option represents an option that can be passed to Register, Open or OpenDB.
+type Option func(*config)
 
-func defaults(cfg *registerConfig) {
+type registerConfig = config
+
+// RegisterOption has been deprecated in favor of Option.
+type RegisterOption = Option
```

Comment:
I think you did it the other way around. The new type is `Option`, and the old one is being deprecated, so please just:
```go
type Option = RegisterOption
```

Reply:
After we did our homework, it turns out the recommendation is `type OldAPI = NewPackage.API`.
Changed in other places for consistency.

Reply:
Good. Thanks for changing elsewhere too. However, I feel I should add that I don't think it's a hard rule and the example you gave is for when a type moves to a different package, which is not the case here.

Reply:
I still feel that your example of the "recommended way" doesn't apply here, because we're not moving packages.